tensorflow/core/common_runtime/next_pluggable_device/next_pluggable_device.cc

```diff
@@ -31,6 +31,9 @@ limitations under the License.
 ABSL_FLAG(bool, next_pluggable_device_use_pjrt_allocator, true,
           "Use PjRtAllocator in next pluggable device.");
 
+ABSL_FLAG(bool, next_pluggable_device_use_pjrt_tensor_buffer, true,
```

Comment:
It doesn't look like this flag is being used? Also, what's the difference between this one and the one above? The description is the same.

Reply:
@swachhandl  thanks for the comment, it is a mistake . it is used when creating the context.

```
device_context_ = core::RefCountPtr<DeviceContext>(new PjRtDeviceContext(
        shape_determination_fns, /*use_pjrt_tensor_buffer=*/absl::GetFlag(
            FLAGS_next_pluggable_device_use_pjrt_tensor_buffer)));
```